spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -32,18 +32,18 @@ In reverse chronological order, here's the specification for each MicroProfile r
 
 [[microprofile3.0]]
 === MicroProfile 3.0 (2Q2019)
-MicroProfile 3.0 is the tenth platform release for the https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
+MicroProfile 3.0 is the next platform release for the https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
```

Comment:
'latest' instead of 'next' ?  And then when we add 3.1, we need a new word again. 
Perhaps use the time 'MP 3.0 is the 2nd platform release in 2019'

Reply:
I was trying to get away from numbering these releases...  Maybe it's okay to promote the number of platform releases?  Should I revert back to "tenth"?  Or, I do like "latest" better than "next".  What do you think?

Reply:
@pilhuhn Oh, now I understand your comment...  If we use "latest" then we have to change it with every release, just like we do with numbering.  I'm thinking about reverting back to numbering...  TBH, another idea I had was to just say "MicroProfile 3.0 is the platform release.."

Reply:
Yes. And for the next release we don't only change the entry of that release, but also of 3.0. 

'MP 3.0 is the 2nd platform release in 2019'  
'MP 3.1 is the 3rd ..'
'MP 3.2 is the 1st platform release in 2020' 

saying '.. is the platform release' sounds odd to me as non-native speaker.

Reply:
I think I'm going to revert back to numbering...  I didn't hear any complaints or suggestions, it was just an attempt to get away from numbering.  But, since we're not coming up with anything better, let's stick with numbering.  Thanks for the discussion!  I'll revert that change, merge, and create an RC2.